src/containers/provider/Provider.js

```diff
@@ -0,0 +1,26 @@
+import { Component, PropTypes } from 'react';
+
+export default class Provider extends Component {
+
+  static childContextTypes = {
+    router: () => {},
+    history: () => {},
+    actions: () => {},
+    action: () => {},
+    state: () => {},
+    store: () => {},
+  };
```

Comment:
I really don't know. This is something copied from `mobx-connect` package. Will investigate.